modules/gltf/gltf_document.cpp

```diff
@@ -465,7 +464,7 @@ Error GLTFDocument::_serialize_nodes(Ref<GLTFState> p_state) {
 		}
 
 		Node *scene_node = nullptr;
-		if (i < scene_node_count) {
+		if (i < (int)p_state->scene_nodes.size()) {
```

Comment:
Odd. Also size() is now int64_t

Reply:
It's not, this is HashMap, which has `uint32_t size()`.